Libraries/PeriphDrivers/Source/CSI2/csi2_ai87.c

```diff
@@ -289,6 +307,23 @@ int MXC_CSI2_PPI_Stop(void)
 /* CSI2 DMA - Used for all features */
 /************************************/
 
+bool MXC_CSI2_DMA_Frame_Complete(void)
+{
+    return MXC_CSI2_RevA_DMA_Frame_Complete();
+}
+
+mxc_csi2_capture_stats_t MXC_CSI2_GetCaptureStats()
+{
+    mxc_csi2_reva_capture_stats_t stats = MXC_CSI2_RevA_DMA_GetCaptureStats();
```

Comment:
Can you just return stats here? Or return the RevA function call? I forget how the casting works for structs.

Reply:
```C
mxc_csi2_capture_stats_t MXC_CSI2_GetCaptureStats()
{
    mxc_csi2_reva_capture_stats_t stats = MXC_CSI2_RevA_DMA_GetCaptureStats();
    return (mxc_csi2_capture_stats_t)stats;
}
```

Gives
```shell
/home/jakecarter/repos/fork/msdk/Libraries/CMSIS/../PeriphDrivers/Source/CSI2/csi2_ai87.c: In function 'MXC_CSI2_GetCaptureStats':
/home/jakecarter/repos/fork/msdk/Libraries/CMSIS/../PeriphDrivers/Source/CSI2/csi2_ai87.c:325:5: error: conversion to non-scalar type requested
  325 |     return (mxc_csi2_capture_stats_t)stats;
      |     ^~~~~~
```

I think we need to return a pointer to do the type cast but I just wanted to return the struct directly.

I'll see if I can clean this up a bit